webapp/templates/base.html

```diff
@@ -98,8 +98,15 @@
                   <br>
                 </div>
               `);
+              $('#water').remove();
```

Comment:
You could have a map of resource names -> values that you pass to jinja. then you loop through.
That will scale better and reduce redundant code.